lib/octokit/client/pull_requests.rb

```diff
@@ -96,11 +96,9 @@ def create_pull_request_for_issue(repo, base, head, issue, options = {})
       # @return [Sawyer::Resource] Hash representing updated pull request.
       # @see https://developer.github.com/v3/pulls/#update-a-pull-request
       # @example
-      #   @client.update_pull_request('octokit/octokit.rb', 67, 'new title', 'updated body', 'closed')
-      # @example Passing nil for optional attributes to update specific attributes.
-      #   @client.update_pull_request('octokit/octokit.rb', 67, nil, nil, 'open')
```

Comment:
`title` is mandatory field. It cannot be `nil`, otherwise.

```
Error summary:
  resource: PullRequest
  code: missing_field
  field: title // See: https://docs.github.com/rest/reference/pulls#update-a-pull-request
```